t/t0030-heartbeat-stonith.t

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
+
+test_description='Check heartbeat stonith script for powerman'
+
+. `dirname $0`/sharness.sh
+
+export pm_sim=$SHARNESS_TEST_SRCDIR/scripts/pm-sim.sh
+export stonith=$SHARNESS_TEST_SRCDIR/../heartbeat/powerman
+
+echo "Command completed successfully" >success.exp
+
+is_successful() {
+	test_cmp success.exp $1
+}
+
+test_expect_success 'reset simulated state' '
+	rm -f pm-sim.data
+'
+test_expect_success 'single plug: stonith on t works' '
+	PM_OVERRIDE=$pm_sim PM_VERBOSE=1 \
+	    $stonith on t >on.out 2>on.err &&
+	is_successful on.out &&
+	cat >on_err.exp <<-EOT &&
+	powerman-stonith: $pm_sim -1 t, rc=0
+	EOT
+	test_cmp on_err.exp on.err
+'
```

Comment:
was able to figure out what was going on, but perhaps a few lines of comments for the uninitiated might be good.  that we're testing that `stonith` is sending powerman client the right stuff and `$pm_sim` is the fake powerman client to get info on what stonith was doing.